src/Symfony/Validator/Metadata/Property/ValidatorPropertyMetadataFactory.php

```diff
@@ -78,13 +73,8 @@ final class ValidatorPropertyMetadataFactory implements PropertyMetadataFactoryI
     /**
      * @param PropertySchemaRestrictionMetadataInterface[] $restrictionsMetadata
      */
-    public function __construct(
-        private readonly ValidatorMetadataFactoryInterface $validatorMetadataFactory,
-        private readonly PropertyMetadataFactoryInterface $decorated,
-        private readonly iterable $restrictionsMetadata = [],
-        ?ContainerInterface $container = null,
```

Comment:
this is a bc but as it was tagged like a week ago and we revert it impacts should be negligible.